app/templates/odkazy.html

```diff
@@ -19,13 +19,15 @@ <h4>Oficiální zdroje</h4>
             <li><a href="https://covid.gov.cz/" target="_blank">Covid portál</a></li>
             <li><a href="https://ockoreport.uzis.cz/" target="_blank">Report vytíženosti očkovacích míst</a></li>
             <li><a href="https://koronavirus.mzcr.cz/wp-content/uploads/2021/03/Navod-k-registraci-a-rezervaci-pro-zajemce-o-ockovani.pdf" target="_blank">Návod pro zájemce o očkování</a></li>
+            <li><a href="https://ockovani.kr-stredocesky.cz/statistika-dodavky-vakcin" target="_blank">Dodávky vakcín do ČR - středočeský kraj</a></li>
```

Comment:
is it official? We should put both their reports to the same group

Reply:
Reordered, put down to have them together. Only gov are now official